README.md

```diff
@@ -106,17 +106,25 @@ Now you have ability to disable elements of Spaceship. All options must be overr
 | Variable | Default | Meaning |
 | :------- | :-----: | ------- |
 | `SPACESHIP_PROMPT_SYMBOL` | `➔` | Character to be shown before any command |
-| `SPACESHIP_PROMPT_VIA` | `via` | Character to be shown before git status |
-| `SPACESHIP_PROMPT_ON` | `on` | Character to be shown before environment version |
 | `SPACESHIP_PROMPT_SEPARATE_LINE` | `true` | Make the prompt span across two lines |
 | `SPACESHIP_PROMPT_ADD_NEWLINE` | `true` | Adds a newline character before each prompt line |
 | `SPACESHIP_PROMPT_TRUNC` | `3` | Number of folders of cwd to show in prompt, 0 to show all |
 
+### Prefixes
```

Comment:
Let's put all prefix-related variables in this section. That will be more predictable. Probably user doesn't expect to see `_PREFIX_` variable among `_GIT_` or `_NVM_` ones, right?